MOPTools/twa_mops/ontospecies.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+from typing import List, Optional
+from twa.data_model.base_ontology import BaseOntology, BaseClass, ObjectProperty, DatatypeProperty
+import om
+import geo
+
+from rdkit.Chem import GetPeriodicTable
+from rdkit.Chem.rdmolfiles import MolFromXYZFile
+
+# NOTE TODO this script is incomplete
```

Comment:
This isn't a very helpful to-do note. Please explain what is missing.

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/1330/commits/2b36b35b9b6d806f9bbabfc6bebf16ee822f2f51